esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -393,8 +393,6 @@ define([
 
         _onSprayXml: function(event) {
             var context = this;
-            this.sprayXmlTargetRowTag = registry.byId(this.id + "sprayXmlTargetRowTag");
-            if (this.sprayXmlTargetRowTag.getValue()) {
```

Comment:
Is sprayXmlTargetRowTag not needed for the call to ESP?  (I am not sure I understand why it was there?